cluster/saltbase/salt/nginx/init.sls

```diff
@@ -28,10 +30,20 @@ nginx:
     - group: root
     - mode: 644
 
+{% if grains.cloud in ['gce'] %}
+/etc/kubernetes/manifests/nginx.json:
+  file:
+    - managed
+    - source: salt://nginx/nginx.json
+    - user: root
+    - group: root
+    - mode: 644
+{% else %}
 nginx-service:
   service:
     - running
     - name: nginx
+{% endif %}
     - watch:
```

Comment:
nginx will not be restarted anymore by kubelet if the watched file changes.
We will need to add logic to do it.